Podfile

```diff
@@ -39,9 +39,7 @@ pod 'ISO8601DateFormatter', '0.7'
 pod 'ARCollectionViewMasonryLayout', '~> 2.0.0'
 pod 'SDWebImage', '~> 3.7'
 pod 'SVProgressHUD'
-pod 'RACAlertAction'
```

Comment:
Moved this to be code inside the project instead. ReactiveCocoa podspec was giving me a hassle.